contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -416,6 +481,8 @@ contract LidoOracle is ILidoOracle, AragonApp {
             .mul(beaconSpec.epochsPerFrame);
         MIN_REPORTABLE_EPOCH_ID_POSITION.setStorageUint256(minReportableEpochId);
         emit MinReportableEpochIdUpdated(minReportableEpochId);
+
+        LAST_COMPLETED_REPORT.setStorageUint256(_getTime() << 128 | modeReport.beaconBalance);  // fixme: I need modeReport.beaconValidators also!
```

Comment:
I am thinking if that would be cheaper to go to `lido.getBeaconStat()` and grab it from there (then to store this on our side)?..

Reply:
I think it won't. Cross apps aragon calls are very expensive